tcs/csp_solver_pc_heat_sink.cpp

```diff
@@ -112,7 +112,7 @@ void C_pc_heat_sink::init(C_csp_power_cycle::S_solved_params &solved_params)
 	}
 
 	// Calculate the design point HTF mass flow rate
-	double cp_htf_des = mc_pc_htfProps.Cp_ave(ms_params.m_T_htf_cold_des+273.15, ms_params.m_T_htf_hot_des+273.15, 5);	//[kJ/kg-K]
```

Comment:
Is the '5' parameter represented in your implementation of cp_ave? I think it represents the number of integration points, which defaults to 1 in the new code?

Reply:
The 5 parameter is the number of integration points in this old code. Correct, the new code defaults to 1 if the HTF properties are internal and not a user table. A value of 1 integration points returns the Cp value at the midpoint which was the original implementation throughout most of the system model. It was unclear to me why this particular instance used 5 integration points... it looks like you added it 7 years ago. https://github.com/NREL/ssc/commit/3793ab574c124d8c4ed1d866e4782b34571c15af